packages/frontend-2/lib/projects/composables/projectManagement.ts

```diff
@@ -133,6 +137,37 @@ export function useCreateProject() {
               },
               { fieldNameWhitelist: ['admin'] }
             )
+
+            // New cache update for Workspace.projects
+            if (input.workspaceId) {
+              const workspaceCacheId = getCacheId('Workspace', input.workspaceId)
+
+              modifyObjectFields<
+                { filter?: { search?: string } },
+                { items: Reference[]; totalCount: number }
+              >(cache, workspaceCacheId, (fieldName, variables, value, details) => {
+                if (fieldName !== 'projects') return
+                if (variables?.filter?.search?.length) return
```

Comment:
I just realized you can do the eviction from here so that you don't have to do the other evictObjectFields call.
```
if (variables?.filter?.search?.length) {
  return details.DELETE
}
```